Cargo.toml

```diff
@@ -170,7 +170,8 @@ log = "0.4"
 loupe = "0.1"
 lru = "0.7.2"
 memmap2 = "0.5"
-memoffset = "0.8"
+memoffset = "0.8.0"
+mmap-rs = "0.6.0"
```

Comment:
Oh, actually, no. I was using mmap-rs because I needed growable files and memmap2 doesn't have anything to support that. But now I don't need files so I may as well go back to memmap2.

Btw though, memmap2 requires two additional unsafe blocks because it doesn't provide a safe API to access the underlying slices. But that's not really a big deal.